src/scripts/configure.ac

```diff
@@ -874,11 +874,11 @@ EOF
       fi
   done
 
-  if test -z $got_opt_whole_archive
-  then
-    echo "Don't know how to build libcharm.so for Charm4py"
-    test_finish 1
-  fi
+  # if test -z $got_opt_whole_archive
+  # then
+    # echo "Don't know how to build libcharm.so for Charm4py"
+    # test_finish 1
+  # fi
```

Comment:
Is commenting this out necessary? This while-archive options seems important.

Reply:
This blocks the build on my machine if it isn't commented out. I haven't determined if this is the source of the setup.py issue where the build doesn't have a commit id.

Reply:
just looked, the whole archive is needed for the build to work, it's fixed (for the cmake build) in the new commit